spec/models/conversation.rb

```diff
@@ -1,4 +1,8 @@
 class Conversation < ActiveRecord::Base
   belongs_to :candidate
+  delegate(
+    :candidate_profile,
+    to: :candidate
+  )
```

Comment:
```suggestion
    has_one :candidate_profile, through: :candidate
```
Does it work if you do this instead?

Reply:
So when trying this on our code base it fixes the issues specific to `counter_culture` but it then breaks a few other things for us so it appears we are actually relying on it being a delegation instead of a `has_one` although I haven't been able to pinpoint exactly why.